### PR TITLE
update the best practices link since the content was moved to sdk

### PR DESCRIPTION
**Pull request by camilamacedo86**
See : https://github.com/operator-framework/community-operators/pull/3491

### CODE REVIEW

File: content/en/docs/best-practices/common.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: Common recommendations and suggestions to distribute solutions with
 
 ## Overview
 
-Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices].
+Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices] and ensure that you check out [Running on Cluster][running-on-cluster].
```
Comment (timflannagan) [inline content/en/docs/best-practices/common.md:10]: Real minor nit, but if we're presumably referencing section titles in the OLM/SDK documentation, should we update `Running on Cluster` -> `Running On-Cluster`?

/lgtm

File: content/en/docs/best-practices/common.md [hunk 2]
```diff
@@ -37,8 +37,9 @@ In the [CSV](/docs/concepts/crds/clusterserviceversion) manifest of your operato
 
 It is recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available, which is not necessarily the case for all projects.
 
-[operator-best-practices]: https://operator-framework.github.io/community-operators/best-practices/
+[operator-best-practices]: https://master.sdk.operatorframework.io/docs/best-practices/best-practices/
```
Comment (anik120) [inline content/en/docs/best-practices/common.md:40]: Is it possible to pin this to a particular releases rather than master?

Comment (timflannagan) [reply content/en/docs/best-practices/common.md:40]: Does OLM have any sense of a SDK compatibility matrix? Generally agree that parking on a specific release is better than master given links can break over time.

File: content/en/docs/best-practices/common.md (hunk 1 above)
Comment (camilamacedo86) [reply content/en/docs/best-practices/common.md]: Done.

File: content/en/docs/best-practices/common.md (hunk 2 above)
Comment (camilamacedo86) [reply content/en/docs/best-practices/common.md:41]: @anik120 we have the latest one which is usually the link used. 
However, this doc is not present there yet. So, I added a todo for we know that we need to update that after the next release.

**Review by timflannagan — APPROVED** [review]

**Comment by timflannagan** [comment]
https://github.com/operator-framework/olm-docs/pull/153 to address the unrelated linting failure.

File: content/en/docs/best-practices/common.md [hunk 3]
```diff
@@ -37,8 +37,10 @@ In the [CSV](/docs/concepts/crds/clusterserviceversion) manifest of your operato
 
 It is recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available, which is not necessarily the case for all projects.
 
-[operator-best-practices]: https://operator-framework.github.io/community-operators/best-practices/
+<!--todo: after the sdk release 1.7.0 replace the following link with sdk.operatorframework.io/docs/best-practices/best-practices/ -->
+[operator-best-practices]: https://master.sdk.operatorframework.io/docs/best-practices/best-practices/
```
Comment (camilamacedo86) [inline content/en/docs/best-practices/common.md:41]:
```suggestion
[operator-best-practices]: https://sdk.operatorframework.io/docs/best-practices/best-practices/
```

Comment (camilamacedo86) [reply content/en/docs/best-practices/common.md:41]: We can change it now.

**Pull request merged**